plugins/yara/src/plugin.cpp

```diff
@@ -606,26 +627,54 @@ class plugin final : public virtual operator_plugin<yara_operator> {
     return {.transformation = true};
   }
 
+  auto make(invocation inv, session ctx) const
+    -> failure_or<operator_ptr> override {
+    auto args = operator_args{};
+    argument_parser2::operator_("yara")
+      // XXX: How do I get a variable number of args? std::vector<std::string>?
+      // .add(args.rules, "<rules>")
+      // XXX: Rename to `compile`?
```

Comment:
As far as I understand this option doesn't compile rules, but instead takes rules previously compiled with `yarac` instead of the plaintext yara rules the `rules` argument takes.